src/main/java/com/three_tech_solutions/slot_app/services/implementations/UserServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package com.three_tech_solutions.slot_app.services.implementations;
+
+import com.three_tech_solutions.slot_app.data.models.User;
+import com.three_tech_solutions.slot_app.data.repositories.UserRepository;
+import com.three_tech_solutions.slot_app.services.interfaces.UserService;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class UserServiceImpl implements UserService {
+    private final UserRepository userRepository;
+
+    public UserServiceImpl(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    @Override
+    public Optional<User> getById(UUID id) {
+        return userRepository.findById(id);
+    }
```

Comment:
Siguiendo el comentario que te dije sobre que el UserService deberia ser el responsable de lanzar la exception, seria algo como esto

```suggestion
    @Override
    public User getUserByIdOrThrowException(UUID id) {
        return userRepository.findById(id)
                .orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "Credenciales ingresadas incorrectas"));
    }
```